config/routes.rb

```diff
@@ -6,9 +6,11 @@
   get "up" => "rails/health#show", as: :rails_health_check
 
   # Defines the root path route ("/")
-  # root "posts#index"
+  root "users#index"
   resources :users, only: [:index, :show] do
```

Comment:
👏 Good progress on addressing the N+1 issues! 👏 However, there are a couple of areas that still need attention:

- [ ] **N+1 Problem in `users#index`:** Although you've made improvements, I noticed that in the `users#index` file, there is still an N+1 issue. For each user, the application is making additional queries. Please consider updating your code to eliminate these extra queries and optimize the loading of associated data.
![image](https://github.com/MohammadYaser/blog-app/assets/55840999/641ff40a-2681-4781-af34-3995fee882af)

- [ ] **N+1 Issue in `users#show`:** In the `users#show` action, the page still exhibits an N+1 issue. Review the log, and you'll observe that for each post, the application is separately counting comments and likes. Please adjust your code to efficiently load this data without triggering multiple queries for each post.
![image](https://github.com/MohammadYaser/blog-app/assets/55840999/d91975ce-d579-4761-b737-bc837692caf9)

Addressing these points will contribute to further optimizing your application and providing a smoother user experience. Keep up the good work! 🚀🛠️